pod-resizer/README.md

```diff
@@ -0,0 +1,29 @@
+# pod-resizer
+
+This container image watches over another container in a deployment, and
+vertically scales the dependent container up and down. Currently the only
+option is to scale it linearly based on the number of nodes, and it only works
+for a singleton.
+
+## Nanny program and arguments
+
+The nanny scales resources linearly with the number of nodes in the cluster. The base and marginal resource requirements are given as command line arguments, but you cannot give a marginal requirement without a base requirement.
```

Comment:
Yeah, like and answer to the question: how do i use this thing for any purpose other than the one it was written for. Generally a good readme will attract a couple of contributors to hack and re-purpose your example for something cool.